com.prateekmedia.appimagepool.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: com.prateekmedia.appimagepool
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+command: appimagepool
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
```

Comment:
do you need `device=all`?

Reply:
I don't know much about this one, I took this from fluffychat's flatpak build.

Reply:
that's a question that need an answer.